src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -58,10 +58,10 @@ static ColliderCylinderInit sCylinderInit = {
 static CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, MASS_IMMOVABLE };
 
 typedef enum {
-    /* 0 */ ENMA1_ANIM_0,
-    /* 1 */ ENMA1_ANIM_1,
-    /* 2 */ ENMA1_ANIM_2,
-    /* 3 */ ENMA1_ANIM_3
+    /* 0 */ ENMA1_ANIM_UNUSED,
```

Comment:
Even though its unused, its still nameable. IDLE_NOMORPH seems fine. if you want to comment on its unused-ness, `// unused` next to it is fine, but dont think its all that necessary personally